### DISCUSSION TITLE
[求助]怎么快速升级

### DISCUSSION THREAD

**Discussion by otmafo** [share121/inter-knot#1510] [其他]
**送一只凌华**

![](https://xatpicture-1319815905.cos.ap-nanjing.myqcloud.com/202408112145702.jpeg)

**Comment by capacitor1** [comment]
这个委托我不接。

**Comment by FurinaShark** [comment]
（复制）等级与你的GitHub项目数量有关
进入GitHub主页
点击右上角加号
创建新项目（需要公开）
然后你就1级了
